src/structstore_containers.hpp

```diff
@@ -112,6 +112,10 @@ class List {
     }
 };
 
+bool operator== (const List& l1, const List& l2) {
```

Comment:
```suggestion
bool operator==(const List& l1, const List& l2) {
```
Maybe remove the comparison operator, see comments below